about/changelog.md

```diff
@@ -8,6 +8,74 @@ keywords: [changelog, upgrades, updates, releases]
+### 🤖 New AI Companion: SQL Assistant
+
+SQL Assistant uses AI to help you write SQL faster and more accurately.
+
+**Real-Time Help:** chat with models like OpenAI 4o and Claude 3.5 Sonnet to get help writing SQL. Describe what you want in natural language and have AI write the SQL for you.
+
+<!-- TODO: embed this YouTube video https://youtu.be/3Droej_E0cQ -->
+
+**Error Resolution**: when you run into an error, we'll propose a recommended fix that you can choose to accept.
+
+![](https://assets.timescale.com/docs/images/ai-error-fix.png)
+
+**Generate Titles and Descriptions**: click a button and we'll generate a title and description for your query. No more untitled queries!
+
+![](https://assets.timescale.com/docs/images/ai-generate-title.png)
+
+See our [blog post](https://www.timescale.com/blog/postgres-gui-sql-assistant/) or [docs](https://docs.timescale.com/getting-started/latest/run-queries-from-console/#sql-assistant) for full details!
+
+### 🏄 TimescaleDB v2.17 - Performance improvements for analytical queries and continuous aggregate refreshes
+
+All new services created on Timescale Cloud starting this week use TimescaleDB v2.17 Existing services are upgraded gradually during their maintenance windows.
+
+TimescaleDB v2.17 significantly improves the performance of continuous aggregate refreshes, and contains performance improvements for analytical queries and delete operations over compressed hypertables.
+
+We recommend upgrading at the next available opportunity.
+
+**Highlighted features in TimescaleDB v2.17**
+
+*   Significant performance improvements for continuous aggregate policies:
+
+*   Continuous aggregate refresh is now using `merge` instead of deleting old materialized data and re-inserting.
+
+*   This update can decrease dramatically the amount of data that must be written on the continuous aggregate in the presence of a small number of changes, reduce the `i/o` cost of refreshing a continuous aggregate, and generate fewer Write-Ahead Logs (`WAL`). Overall, continuous aggregate policies will be more lightweight, use less system resources, and complete faster.
+
+*   Increased performance for real-time analytical queries over compressed hypertables:
+
+*   We are excited to introduce additional Single Instruction, Multiple Data (`SIMD`) vectorization optimization to our engine by supporting vectorized execution for queries that group by using the `segment_by` column(s) and aggregate using the basic aggregate functions (`sum`, `count`, `avg`, `min`, `max`).
+
+*   Stay tuned for more to come in follow-up releases! Support for grouping on additional columns, filtered aggregation, vectorized expressions, and `time_bucket` is coming soon.
+
+*   Improved performance of deletes on compressed hypertables when a large amount of data is affected.
+
+*   This improvement speeds up operations that delete whole segments by skipping the decompression step. It is enabled for all deletes that filter by the `segment_by` column(s).
```

Comment:
```suggestion
    This improvement speeds up operations that delete whole segments by skipping the decompression step. It is enabled for all deletes that filter by the `segment_by` column(s).
```
This is a follow in sentence from the previous point rather than a new point.